content/docker/docker-desktop-troubleshooting/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+content_type: topic
+prerequisites:
+  hard:
+  - docker/intro-to-docker
+ready: false
+title: Troubleshooting Docker Desktop for Windows
+---
+
+Docker on Windows can be quite tricky to setup and get working, and a number of errors can occur when trying to get it up and running. In this topic, we will try and provide solutions to some of the problems you may encounter when dealing with Docker Desktop.
+
+## Hyper-V
+Docker Desktop requires Hyper-V to be installed as well as enabled on your machine. This installation is in the form of a feature. Click [here](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v) and follow the instructions to ensure Hyper-V is enabled on your machine.
+
+### Help! I dont see Hyper-V on my list of Programs and features!
```

Comment:
inconsistent capitalisation. Thy does Programs get a capital and features doesn't?

Just use regular sentence-case in the headings.